woocommerce/woocommerce-admin
# Make WooCommerce-admin full-screen minimum height 100vh important

Fixes #6372 

Make WooCommerce-admin full-screen minimum height 100vh important.


### Screenshots
Before
<img width="543" alt="螢幕快照 2021-06-23 下午4 26 00" src="https://user-images.githubusercontent.com/56378160/123166573-90131880-d443-11eb-869f-2f34828af5d8.png">

After
<img width="527" alt="螢幕快照 2021-06-23 下午4 26 26" src="https://user-images.githubusercontent.com/56378160/123166591-9608f980-d443-11eb-900d-46664a7dcbe1.png">


### Detailed test instructions:

1. Use Safari and make browser height less than 615px.
1. Go to WooCommerce home page -> Onboarding wizard "Business details" 
1. Click on How many products do you plan to display? and Currently selling elsewhere? dropdowns.
1. Observe no gap on the bottom
